cmd/webhook/main.go

```diff
@@ -286,6 +294,10 @@ func NewConversionController(ctx context.Context, cmw configmap.Watcher) *contro
 }
 
 func main() {
+	sbSelector := psbinding.WithSelector(psbinding.ExclusionSelector)
+	if os.Getenv("SINK_BINDING_OPT_OUT_SELECTOR") == "true" {
+		sbSelector = psbinding.WithSelector(psbinding.InclusionSelector)
```

Comment:
if opt out is true... opt in? wat?

Reply:
The `Inclusion` refers to the selector operator, while it's effect is to `Opt Out` of the webhook by default (you have opt in by adding `bindings.knative.dev/include:true` label).  

I blindly copied the name from a previous PR and I admit this is confusing :)  Please take a look at #2662 and see if it's clearer.